src/app/screens/Onboard/NewWallet/index.js

```diff
@@ -80,7 +126,11 @@ export default function ConnectLndHub() {
                 return false;
               }}
             />
-            <Button label="Continue" onClick={handleNext} primary />
+            <Button
+              label={lndHubData.login ? "Next" : "Create a wallet"}
+              onClick={handleNext}
```

Comment:
You can directly pass in the right func to onClick, so there is no handleNext method needed.
`onClick={lndHubData.login && lndHubData.password ? next : signup}`